Tiltfile.dev

```diff
@@ -0,0 +1,18 @@
+local_resource(
+  name = 'dev:strike-off-objections-api-build',
+  cmd = 'mvn compile',
+  deps = ['src']
+)
+
+custom_build(
+  ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/strike-off-objections-api',
+  command = 'mvn compile jib:dockerBuild -Dimage=$EXPECTED_REF',
+  live_update = [
+    sync(
+      local_path = './target/classes',
+      remote_path = '/app/classes'
+    ),
+    restart_container()
+  ],
+  deps = ['./target/classes']
+)
```

Comment:
Newline.

Reply:
Happy to add a new line, but what does the newline provide? Why is it needed

Reply:
I think sonar complains about it; github always does.

Reply:
Sonar didn't raise it as a smell, bug, hotspot or vulnerability so I think Sonar is ok with it.
And the Red circle on Github is less complaining and more letting you know. I am yet to ever encounter an issue caused by this

Reply:
In that case it's just a styling preference rather than anything else.